functions/New-DbaAvailabilityGroup.ps1

```diff
@@ -246,7 +256,7 @@ function New-DbaAvailabilityGroup {
         [ValidateSet('None', 'Primary', 'Secondary', 'SecondaryOnly')]
         [string]$AutomatedBackupPreference = 'Secondary',
         [ValidateSet('OnAnyQualifiedFailureCondition', 'OnCriticalServerErrors', 'OnModerateServerErrors', 'OnServerDown', 'OnServerUnresponsive')]
-        [string]$FailureConditionLevel = "OnServerDown",
+        [string]$FailureConditionLevel = (Get-DbatoolsConfigValue -FullName 'AvailabilityGroups.Default.FailureConditionLevel' -Fallback 'OnCriticalServerErrors'),
```

Comment:
I sure hope release notes have a big caveat as this being a breaking change.

Reply:
I also updated the DESCRIPTION